bin/plugin/open/selfAddIngressKey

```diff
@@ -67,14 +67,20 @@ if (not defined $pubKey) {
     my $algos    = join(' ', @algoList);
     osh_info "Please paste the SSH key you want to add. This bastion supports the following algorithms:\n";
 
+    if (grep { 'ed25519-sk' eq $_ } @algoList) {
+        osh_info "ED25519 Secure-Key: strongness[######] speed[#####], use `ssh-keygen -t ed25519-sk' to generate one";
+    }
     if (grep { 'ed25519' eq $_ } @algoList) {
-        osh_info "ED25519: strongness[#####] speed[#####], use `ssh-keygen -t ed25519' to generate one";
+        osh_info "ED25519: strongness[#####.] speed[#####], use `ssh-keygen -t ed25519' to generate one";
+    }
+    if (grep { 'ecdsa-sk' eq $_ } @algoList) {
+        osh_info "NIST-P Secure key: strongness[#####.] speed[#####], use `ssh-keygen -t ecdsa-sk -b 521' to generate one";
     }
     if (grep { 'ecdsa' eq $_ } @algoList) {
-        osh_info "ECDSA  : strongness[####.] speed[#####], use `ssh-keygen -t ecdsa -b 521' to generate one";
+        osh_info "ECDSA  : strongness[####..] speed[#####], use `ssh-keygen -t ecdsa -b 521' to generate one";
     }
     if (grep { 'rsa' eq $_ } @algoList) {
-        osh_info "RSA    : strongness[###..] speed[#....], use `ssh-keygen -t rsa -b 4096' to generate one";
+        osh_info "RSA    : strongness[###...] speed[#....], use `ssh-keygen -t rsa -b 4096' to generate one";
```

Comment:
This whole block should be moved somewhere else as a function, because we're using similar things in several plugins. It's used in:

```
bin/plugin/open/selfAddIngressKey
bin/plugin/restricted/accountCreate
bin/plugin/restricted/groupCreate
bin/plugin/restricted/realmCreate
```

This is not directly linked to this PR, but as you're modifying it, the change would need to be done in all the above plugins. A cleaner way would be to add a func e.g. `print_algorithms_hint` in `lib/ssh.inc` that would autodetect which algorithms are supported on current system (by matching the OpenSSH version as already done here), plus matching either `allowedIngressSshAlgorithms` or `allowedEgressSshAlgorithms` configuration options of `bastion.conf` to hide disallowed algorithms even if the system supports them. The only argument to this func would be to tell it whether it's supposed to print info for ingress or egress keys, to fetch the proper `bastion.conf` option.

Reply:
If going this way, `print_algorithms_hint` needs to be declared in the `%_autoload_files`/`ssh` array in `Bastion.pm`, so the `ssh.inc` file is autoloaded in JIT if this func is called.